sdk/cosmosdb/cosmos/package.json

```diff
@@ -135,5 +135,15 @@
     "ts-node": "^9.0.0",
     "typedoc": "0.15.2",
     "typescript": "~4.2.0"
+  },
+  "//sampleConfiguration": {
+    "skipFolder": true,
```

Comment:
`skipFolder` may not be quite enough. Based on what I'm seeing in CI, we probably need to use:

```
"skip": [
  "<filename>.ts"
]
```